metadata-ingestion/src/datahub/ingestion/source/grafana/field_utils.py

```diff
@@ -0,0 +1,200 @@
+import logging
+from typing import Any, Dict, List, Union
+
+from datahub.ingestion.source.grafana.models import Panel
+from datahub.metadata.schema_classes import (
+    NumberTypeClass,
+    SchemaFieldClass,
+    SchemaFieldDataTypeClass,
+    StringTypeClass,
+    TimeTypeClass,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def _deduplicate_fields(fields: List[SchemaFieldClass]) -> List[SchemaFieldClass]:
```

Comment:
there is already some deduplication for schema fields 
https://github.com/datahub-project/datahub/blob/e9df5401cbf5c3dd8bc0b7cf3377cf4c87c3c371/metadata-ingestion/src/datahub/ingestion/api/source.py#L456